src/client/pages/ToeristischeVerhuur/ToeristischeVerhuur.tsx

```diff
@@ -289,20 +285,12 @@ export default function ToeristischeVerhuur() {
           name: 'Datatabel',
         }}
       >
-        {vergunningen.some((vergunning) => vergunning.isActual) && (
+        {vergunningen && (
           <Table
             className={styles.TableVergunningen}
             titleKey="title"
             displayProps={DISPLAY_PROPS_VERGUNNINGEN}
-            items={vergunningen.filter((vergunning) => vergunning.isActual)}
-          />
-        )}
-        {vergunningen.some((vergunning) => !vergunning.isActual) && (
-          <Table
-            className={styles.TableVergunningen}
-            titleKey="title"
-            displayProps={DISPLAY_PROPS_VERGUNNINGEN_EXPIRED}
-            items={vergunningen.filter((vergunning) => !vergunning.isActual)}
+            items={vergunningen}
```

Comment:
Ideally the vergunningen should be sorted by Status somehow. You'd want your most actual items on top.

So something like:
- First aanvragen (no decision yet) by date request asc
- Actual vergunningen sorted by date end asc
- Verlopen vergunningen sorted by date end Desc

But.. having both B&B + Verkantieverhuur is edge-case'ey. NVM

Reply:
I can sort on date request?

Reply:
I guess that should cover at least the sorting of one type of vergunning